cli/tests/tui/test_executor.py

```
@@ -306,6 +306,30 @@ async def test_redirected_subprocess_output_is_decoded_as_utf8() -> None:
+@pytest.mark.asyncio
+async def test_pipe_executor_streams_prompt_without_waiting_for_newline() -> None:
+    executor = CommandExecutor(use_pty=False)
+    output: list[str] = []
+
+    async for event in executor.run(
+        sys.executable,
+        (
+            "-u",
+            "-c",
+            "import sys; sys.stdout.write('Select: '); sys.stdout.flush(); "
+            "answer=sys.stdin.readline().strip(); print('picked ' + answer)",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)